Mathlib/Topology/Algebra/ConstMulAction.lean

```diff
@@ -474,19 +494,18 @@ section MulAction
 variable {G₀ : Type*} [GroupWithZero G₀] [MulAction G₀ α] [TopologicalSpace α]
   [ContinuousConstSMul G₀ α]
 
--- Porting note: generalize to a group action + `IsUnit`
```

Comment:
I think this should be kept as a TODO about `IsUnit`

Reply:
Restored.